cfme/tests/containers/test_relationships_tables.py

```diff
@@ -3,10 +3,15 @@
 import pytest
 from cfme.fixtures import pytest_selenium as sel
 from cfme.containers import list_tbl as list_tbl_pods
+from cfme.containers import list_tbl as list_tbl_pods_rel
 from cfme.containers import list_tbl as list_tbl_services
+from cfme.containers import list_tbl as list_tbl_services_rel
 from cfme.containers import list_tbl as list_tbl_nodes
+from cfme.containers import list_tbl as list_tbl_nodes_rel
```

Comment:
why do we import the same name dozens of times as a different name, something seems wrong there